### ISSUE TITLE
Licence of code?

### ISSUE THREAD

**Issue by hokreb** [minikeyness/condorweb#1]
Hello Minikeyness,

I found your interesting web project interfacing condor. I am on the search for something similar, but with english interface. What kind of licence do you apply to the code? Is it ok that I reuse extend this code?

best regards
  nhok

**Comment by minikeyness** [comment]
of course, it is ok. This is just like a demo which don't have actual function.
